### ISSUE TITLE
Wrong copyright information

### ISSUE THREAD

**Issue by madewild** [ec-doris/kohesio.eu#106]
A lot of images (including the placeholder when no image is present) have the copyright info "“©Société de Gestion de la Station des Rousses, SAEM SOGESTAR / Benjamin Becker (2017)”.
This seems to be an error, we need to check what to display (if anything at all) for generic images.

**Comment by madewild** [comment]
@teodoracostin Thierry was asking
Could you clarify 
-	what is the copyright information of the logos, and 
-	whether or not we have to display it down the logos if we use those logos for replacing missing real pictures in the project details ?

I don't think @raphdom can answer these questions, this is about the origin of the images we use. They come from InfoRegio if I'm not mistaken so we should cross-check to display the correct copytight info.

**Comment by D063520** [comment]
Basically the copyright of inforregio is in the knowledge graph. So they are send via API, the other are coming from the front-end . In fact I would prefer to move all the data (including the logos of the thematic objectives) to the knwoledge graph and send you evrything via API .....

**Comment by madewild** [comment]
Yes I agree it would be better to have everything in the knowledge graph. But we first need to get the data right :)

**Comment by madewild** [comment]
Moving to backend
